admin/bin/qserv-configure.py

```diff
@@ -318,11 +321,11 @@ def run(self):
             _LOG.info("Store data in: %s" % self._qserv_data_dir)
             self._templater.applyOnce(in_meta_config_file, self._meta_config_file, params_dict)
 
-        ###################################
+        # ---------------------------------
```

Comment:
You are meant to have a space after the comment character so this change is correct.